src/App/Container.php

```diff
@@ -119,13 +119,13 @@ public function __construct(string $toolsBasePath, string $magentoBasePath)
             \Magento\MagentoCloud\Config\Database\MergedConfig::class
         );
 
-        $this->container->when(DeployProcess\InstallUpdate\ConfigUpdate\Urls::class)
-            ->needs(ProcessInterface::class)
+        $this->container->when(DeployStep\InstallUpdate\ConfigUpdate\Urls::class)
```

Comment:
Why this configuration still in the Contaner file? Looks like it should be configured in scenario XML file.

Reply:
@shiftedreality Is it possible configure steps for sub-steps in senario.xml files?

Reply:
It's currently too complicated for XML file, we'll have to refactor this case later